.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 node_modules/
 test/coverage
+coverage
```

Comment:
@quarkus can we also configure the test coverage to be placed in test/coverage as before?

Reply:
check